specs/disciplina/witness/api/explorer.yaml

```diff
@@ -0,0 +1,215 @@
+          schema:
+            $ref: "#/definitions/ErrResponse"
+        404:
+          description: Address not found
+  /transactions/{transactionHash}:
+    get:
+      summary: >-
+        Get information about a transaction.
+        `txType` field of the result object will denote the type of transaction: "money" or "publication".
+        The corresponding key in the object will contain the rest of information about the transaction.
+      operationId: getTransaction
+      produces: 
+      - application/json
+      parameters:
+      - in: path
+        name: transactionHash
+        description: Hash of the transaction being requested.
+        required: true
+        type: string
+        format: byte
+      responses:
+        200:
+          description: Successful operation
+          schema:
+            $ref: "#/definitions/TransactionInfo"
+        400:
+          description: Invalid input
+          schema:
+            $ref: "#/definitions/ErrResponse"
+        404:
+          description: Transaction not found
+
+definitions:
+  Hash:
+    type: string
+    format: byte
+  Address:
+    type: string
+    format: byte
+  Coin:
+    type: integer
+    format: int64
+  TransactionPart:
+    type: object
+    properties:
+      address:
+        $ref: "#/definitions/Address"
+      value:
+        $ref: "#/definitions/Coin"
+  BlockInfo:
+    type: object
+    properties:
+      headerHash:
+        $ref: "#/definitions/Hash"
+      issuer:
+        type: string
+        format: byte
+      difficulty:
+        type: integer
+        format: int64
+      slotId:
+        type: integer
+        format: int64
+      prevHash:
+        $ref: "#/definitions/Hash"
+      transactions:
+        type: array
+        items:
+          $ref: "#/definitions/TransactionInfo"
+  AddressInfo:
+    type: object
+    properties:
+      balance:
+        type: integer
+        format: int32
+      transactions:
+        type: array
+        items:
+          $ref: "#/definitions/TransactionInfo"
+  TransactionInfo:
+    type: object
+    discriminator: txType
+    properties:
+      txType:
+        type: string
+        enum:
+        - money
+        - publication
+      money:
+        type: object
+        properties:
+          in:
+            $ref: "#/definitions/TransactionPart"
+          outs:
+            type: array
+            items:
+              $ref: "#/definitions/TransactionPart"
+      publication:
+        type: object
+        properties:
+          author:
+            $ref: "#/definitions/Address"
+          block:
+            $ref: "#/definitions/Hash"
+          prevBlock:
+            $ref: "#/definitions/Hash"
+  ErrResponse:
+    type: object
+    required:
+      - error
+    properties:
+      error:
+        type: string
+        enum:
+          - InvalidFormat
+        description: >-
+          Error type:
+            * `InvalidFormat` - Failed to deserialise one of parameters.
```

Comment:
I do not insist on having this type, but it's expected that you either enlist all errors on none, isn't it? ))